### PR TITLE
Fix toc 

### PR DESCRIPTION
**Pull request by xuechunL**
for website docs sidebar.

### CODE REVIEW

**Comment by xuechunL** [comment]
@queenypingcap PTAL

File: README.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-+ [Quick Start](https://github.com/pingcap/tidb/blob/master/docs/QUICKSTART.md)
-+ Step 1: [Download](op-guide/binary-deployment.md#download-and-decompress-the-official-binary-package)
+- [Quick Start](https://github.com/pingcap/tidb/blob/master/docs/QUICKSTART.md)
+- [Step 1: Download](op-guide/binary-deployment.md#download-and-decompress-the-official-binary-package)
 + Step 2: Deploy
   - [Deployment Recommendation](op-guide/recommendation.md)
-  - Ansible Deployment
+  - [Ansible Deployment](op-guide/ansible-deployment.md)
   - [Binary Deployment](op-guide/binary-deployment.md)
   - [Docker Deployment](op-guide/docker-deployment.md)
```
Comment (queenypingcap) [inline README.md]: Please add 
[Cross-Region Deployment](op-guide/location-awareness.md)

File: README.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
   - [TiDB SQL Grammar](sql/README.md)
   - [Compatibility with MySQL](op-guide/mysql-compatibility.md)
 + Step 5: Monitor
   - [Overview of the Monitoring Framework](op-guide/monitor-overview.md)
   - [Monitoring a TiDB Cluster](op-guide/monitoring-tidb.md)
   - [PD Command-Line Tools](op-guide/pd-control.md)
-
 + Advanced Usage
   - [Data Migration from MySQL to TiDB](op-guide/migration.md)
   - [Loader Instructions](./tool/loader.md)
```
Comment (queenypingcap) [inline README.md]: Please remove the following item
- [Loader Instructions](./tool/loader.md)

File: README.md (hunk 1 above)
Comment (xuechunL) [inline README.md]: done, PTAL

File: README.md (hunk 2 above)
Comment (xuechunL) [inline README.md]: fix the link path. PTAL

**Comment by queenypingcap** [comment]
LGTM

**Pull request merged**